docs/services/data/_index.md

```diff
@@ -103,56 +105,57 @@ Export data with the Viam CLI and download your data for offline access.
 
 ### Upload a batch of data
 
-Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app.
+Upload data to the Viam app from your computer or mobile device using the data client API, Viam CLI, or Viam mobile app.
```

Comment:
two comments here:

1) subtle distinction that the user is not uploading data to the Viam app, they're uploading it to the Viam Cloud which the app pulls from like any other client would. 

2) Although it's true one can capture data directly from the mobile app right now, this is a bit of an edge case and I'm not certain we want to be pushing users to do it from our docs. (Not saying we don't, but wanted to check if you all have an explicit reason _to_ mention it here. If not, my 2 cents is that we remove for now until there is a specific reason to include it).

Reply:
on (1)

this is the language we use in static/include/services/apis/generated/data_manager-table.md which is more appropriate:

"
| [`Sync`](/services/data/capture-sync/#sync) | **Sync data stored on the machine to the cloud.** | <p class="center-text"><i class="fas fa-times" title="no"></i></p> |"

Reply:
^ same comment as above; technically saying "Upload data to the Viam app" isn't incorrect if you interpret "Viam app" to mean the monorepo, but for sake of clarity for end user i recommend we adopt the language used in my above comment "Upload data to the Viam cloud.."

Reply:
```suggestion
Upload data to the Viam cloud from your computer or mobile device using the data client API, the Viam CLI, or the Viam mobile app.
```
Okay I'll change it to the Viam cloud. However, the reason the Viam mobile app is here is that this is talking about uploading a batch of data, NOT capturing data.